tasks/fork/maintain-rate.ts

```diff
@@ -44,8 +49,9 @@ HyperdriveDeployBaseTask(
                 }))!,
             );
             let stethBalanceIncrease =
-                (stethCurrentBalance * BigInt(interval)) /
-                (365n * 24n * rateFraction);
+                (stethCurrentBalance * BigInt(interval) * parseEther(rate)) /
+                BigInt(1e18) /
+                (365n * 24n);
```

Comment:
Can we just do `BigInt(1e18) * 365n * 24n`? I think that's clearer than dividing twice personally

Reply:
Totally, agreed